extensions/sql-database-projects/src/models/options/deployOptionsModel.ts

```diff
@@ -219,39 +28,44 @@ export class DeployOptionsModel {
 	 */
 	public getOptionsData(): string[][] {
```

Comment:
more descriptive comment on what the function is returning please. What does the string[][] represent?